hedera-base/src/test/java/com/openelements/hedera/base/test/ProtocolLayerDataCreationTests.java

```diff
@@ -713,4 +714,17 @@ void testFileDeleteRequestCreation(){
         Assertions.assertThrows(NullPointerException.class, () -> new FileDeleteRequest(null, transactionValidDuration, null));
         Assertions.assertThrows(NullPointerException.class, () -> new FileDeleteRequest(null, null, null));
     }
+
+    @Test
+    void testTopicSubmitMessageResultCreation() {
+
+        //given
+        final TransactionId validTransactionId = TransactionId.fromString("0.0.12345@1");
```

Comment:
@IamLRBA The `TransactionId` string passed in `TransactionId.fromString("0.0.123451")` does not match the expected format. Replace the above line with somethinng like `final TransactionId validTransactionId = TransactionId.fromString("0.0.123451@1697590800.123456789");`

Reply:
Thank you for that insight @Ndacyayisenga-droid